File: week-02/day-03/string_04.py
# When saving this quote a disk error has occured. Please fix it.
# Add "always takes longer than" between the words "It" and "you"

quote = "Hofstadter's Law: It you expect, even when you take into account Hofstadter's Law."
solution = quote.split(" ")
print(solution.index("It"))
solution.insert(3, "always takes longer than")
#solution [:3] += ["always takes longer than"] 2.solution
quote = " ".join(solution)

#solution.append("always takes longer than")
#
print(quote)

File: week-07/day-02/string04.js
'use strict';
// When saving this quote a disk error has occured. Please fix it.
// Add "always takes longer than" to between the words "It" and "you"

var quote = "Hofstadter's Law: It you expect, even when you take into account Hofstadter's Law."

let lookFor = "It you";
let plusString = "always takes longer than"

quote = quote.replace(lookFor, "It " + plusString + " you")



console.log(quote)

